db/migrate/20190113103136_create_peeps.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePeeps < ActiveRecord::Migration[5.2]
+  def change
+    create_table(:peeps) do |p|
+      p.belongs_to :user, index: true
+      p.column(:title, :string)
+      p.column(:content, :string)
+      p.column(:user_id, :integer)
+      p.column(:created_at, :datetime)
+    end
+  end
+end
```

Comment:
Loving that you used Active Record, good job. Using a separate migration to add a column is a new concept to me, thanks for the introduction.